### PR TITLE
Extend input device documentation to set correct ubuntu permissions

### LINKED ISSUE #61: Expand march_input_device to fix a permission error
https://github.com/esp8266/source-code-examples/issues/26

### PR DESCRIPTION
**Pull request by Ishadijcks**
Closes #61

### CODE REVIEW

File: doc/march_packages/march_input_device.rst [hunk 1]
```diff
@@ -29,6 +29,24 @@ Receiving messages
...
+
+.. code::
+
+   Cannot open /dev/ttyUSB0: Permission denied
+
+To make sure your user is allowed to access the port, add your user to the required groups:
+
+.. code::
+
+  sudo usermod -a -G tty <your_username>
```
Comment (JitskedeVries) [inline doc/march_packages/march_input_device.rst:44]: Is this your github username or the username of the computer? Might be useful to mention

File: doc/march_packages/march_input_device.rst [hunk 2]
```diff
@@ -29,6 +29,24 @@ Receiving messages
...
+  sudo usermod -a -G dialout <your_username>
+
+Now log out and log back in, the error should be resolved.
```
Comment (JitskedeVries) [inline doc/march_packages/march_input_device.rst]: log out of your computer?

**Review by martijnvandermarel — APPROVED** [review]
Looks good

**Review by JitskedeVries — DISMISSED** [review]
Since I know nothing about the input device I had a question about some instructions. I think it might be helpful to adjust to help input device noobs like me.

File: doc/march_packages/march_input_device.rst (hunk 2 above)
Comment (Ishadijcks) [reply doc/march_packages/march_input_device.rst]: Yes

File: doc/march_packages/march_input_device.rst (hunk 1 above)
Comment (Ishadijcks) [reply doc/march_packages/march_input_device.rst:44]: The name of your ubuntu user

**Comment by Ishadijcks** [comment]
These instructions have nothing to do with the input device, only Ubuntu.

File: doc/march_packages/march_input_device.rst (hunk 2 above)
Comment (Ishadijcks) [inline doc/march_packages/march_input_device.rst]:
```suggestion
Now log out on your computer and log back in, the error should be resolved.
```

File: doc/march_packages/march_input_device.rst (hunk 1 above)
Comment (Ishadijcks) [reply doc/march_packages/march_input_device.rst:44]: I don't think there should be any confusion as to which user this refers to.

**Pull request merged**